### ISSUE TITLE
OIDC graph calls cannot be made in GitHub Codespaces 

### ISSUE THREAD

**Issue by BALAGA-GAYATRI** [Azure/azure-cli#21025]
Customers are facing the issue while making graph API calls in GH- codespaces.
For more details  - https://github.com/Azure/login/issues/177

**Comment by yonzhan** [comment]
Is it related to Azure CLI?

**Comment by jiasli** [comment]
@jongio, is `az login` able to launch a browser? Or did you use device code login?

**Comment by jongio** [comment]
Yes, it can launch a browser, but the codespaces needs to have port 8400 open for that to work.

It should work with either approach but doesn't because the codespace isn't registered valid device.
